### PR TITLE
Adding information about changes in config_dev.yml regarding MySQL

### PR DESCRIPTION
**Pull request by jzawadzki**
Hi!

As described in #309, this pull request adds minor info about additional task required for using MySQL instead of SQLite.

I'm not sure if wording is correct and/or clear enough, so please leave feedback! 
Thanks,
JZ

### CODE REVIEW

File: app/config/parameters.yml.dist [hunk 1]
```diff
@@ -12,6 +12,8 @@ parameters:
     #
     # database_url: mysql://root:pass@127.0.0.1:3306/symfony_demo
     #
+    # Furthermore, you must remove or comment out lines 11-14 from config_dev.yml regarding SQLite!
```
Comment (voronkovich) [inline app/config/parameters.yml.dist]: I think it's better to write: `Furthermore, you must remove or comment out the "doctrine" section from config_dev.yml regarding SQLite!`

Comment (jzawadzki) [inline app/config/parameters.yml.dist]: you're right - changed!

**Comment by bocharsky-bw** [comment]
:+1:

**Comment by xabbuh** [comment]
:+1:

**Comment by stof** [comment]
:+1:

**Comment by javiereguiluz** [comment]
@jzawadzki thanks for this contribution and I'm sorry it took me so long to merge it.

**Pull request closed without merging**